partcad/src/partcad/part_factory_enrich.py

```
@@ -19,50 +19,62 @@
-                if source_project is None:
-                    pc_logging.debug("Available projects: %s" % str(sorted(list(ctx.projects.keys()))))
+                self.source_project = self.ctx.get_project(self.source_project_name)
+                if self.source_project is None:
+                    pc_logging.debug("Available projects: %s" % str(sorted(list(self.ctx.projects.keys()))))
                     raise Exception("Package not found: %s" % self.source_project_name)
-                augmented_config = source_project.get_part_config(self.source_part_name)
+                augmented_config = self.source_project.get_part_config(self.source_part_name)
             if augmented_config is None:
-                pc_logging.error("Failed to find the part to enrich: %s" % self.source_part_name)
+                pc_logging.error(
+                    f"Failed to find the part to enrich: {self.source_project.name}:{self.source_part_name}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)